crates/solidity/outputs/npm/crate/src/generated/parsers.rs

```diff
@@ -8805,7 +8814,7 @@ impl Language {
             if elements.is_empty() {
                 break result;
             }
-            reduce_pratt_elements(&mut elements);
+            reduce_pratt_elements(|children| children, &mut elements);
```

Comment:
thoughts about using `Option<Function>` and passing `Non` in this case?

Reply:
The typing was simpler, and I preferred to not have the conditional logic in the helper